### DISCUSSION TITLE
Export bin schedule to www folder within hass

### DISCUSSION THREAD

**Discussion by j0nnymoe** [robbrad/UKBinCollectionData#458] [Ideas]
### Home Assistant Version

2023.11.2

### Installation Method

Using HACS

### Issue Information

Hey,

Looking to see if it's possible to make the hass component export the json file to `/config/www` so that it can be picked up by hacs_waste_collection? 

Have looked through issues and documentation but couldn't see anything about this. (Apologies if I missed it)

Thanks

### Verification

- [X] I searched for similar issues at https://github.com/robbrad/UKBinCollectionData/issues?q=is:issue and found no duplicates
- [X] If trying to add a specific council, I've checked it is listed as working at https://robbrad.github.io/UKBinCollectionData/3.11/
- [X] I have provided a detailed explanation of the issue as well as steps to replicate the issue
- [X] I understand that this project is run by volunteer contributors therefore completion of this issue cannot be guaranteed

**Comment by j0nnymoe** [comment]
Hey, Is this something that would be possible to add to the hass integration?

Thanks

**Reply by robbrad** [reply]
Can you raise this as an issue and add the tag idea please?

**Comment by robbrad** [comment]
@j0nnymoe : out of interest why would you not just use the hacs custom_component from this repo ?

Is there different functionality in hacs_waste_collection

**Reply by j0nnymoe** [reply]
Hey @robbrad,

Sorry for the late reply. I'm already using the hacs custom_component for this but was hoping to use the hacs_waste_collection to show the data within the calendar.

Cheers

**Comment by sym0nd0** [comment]
I was just reading https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/ukbcd.md and had been on the search for the .json file mentioned. My hope is to feed the data from this custom_component as the source in to hacs_waste_collection_schedule, so that the HA calendar is populated with bin collection dates, for a at a glance forward look and possibly push notifications (the latter I've just thought of as I type this).

Apologies if I can already achieve that from this component alone, my much preferred option, I've been back and forth through the docs.

**Comment by whatdaybob** [comment]
I tried this using the selenium and api docker but unfortunately the output appears to be escaped JSON so we cant use that either, I was hoping that it'll work with the built in downloader of HA but no joy. @sym0nd0 did you find a work around?

**Reply by sym0nd0** [reply]
Thanks to @jamesmacwhite, I've been using this to feed everything and it's been flawless. https://www.gbcbincalendars.co.uk/

Edit, although I realise now that this isn't the thread I thought it was and the link above might not be relevant to you. Sorry.
